Custom build command fails: "error: call to undeclared function 'SetProcessDpiAwareness'"

When attempting to compile tinyfiledialogs, the custom build script fails with:

```
libtinyfiledialogs/tinyfiledialogs.c:1920:2:
error: call to undeclared function 'SetProcessDpiAwareness';
ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
 1920 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
      |         ^
```

It appears this was introduced in #37.

**System Specifications:**

- Windows 10
- rustc v1.77.1
- clang v17.0.6
- MSVC 14.38.33130
- Visual Studio 17 2022

```
[0/7] C:\WINDOWS\system32\cmd.exe /C "cd /D Z:\dev\foo\project\rust\cli && "C:\Program F...l --target-dir Z:/dev/foo/project/build/./cargo/build -- -Cdefault-linker-libraries=yes  
   Updating crates.io index
   Compiling tinyfiledialogs v3.9.1
The following warnings were emitted during compilation:

warning: tinyfiledialogs@3.9.1: clang version 17.0.6
warning: tinyfiledialogs@3.9.1: Target: x86_64-pc-windows-msvc
warning: tinyfiledialogs@3.9.1: Thread model: posix
warning: tinyfiledialogs@3.9.1: InstalledDir: C:\Program Files\LLVM\bin
warning: tinyfiledialogs@3.9.1:  (in-process)
warning: tinyfiledialogs@3.9.1:  "C:\\Program Files\\LLVM\\bin\\clang.exe" -cc1 -triple x86_64-pc-windows-msvc19.38.33135 -emit-obj -mrelax-all -mincremental-linker-compatible -disable-free -clear-ast-before-backend -disable-llvm-verifier -discard-value-names -main-file-name tinyfiledialogs.c -mrelocation-model pic -pic-level 2 -mframe-pointer=all -fmath-errno -ffp-contract=on -fno-rounding-math -mconstructor-aliases -funwind-tables=2 -target-cpu x86-64 -tune-cpu generic -gno-column-info -gcodeview -debug-info-kind=constructor -v -ffunction-sections -fdata-sections "-fcoverage-compilation-dir=C:\\Users\\My Name\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\tinyfiledialogs-3.9.1" -resource-dir "C:\\Program Files\\LLVM\\lib\\clang\\17" -D USE_WINDOWS_HIDPI -internal-isystem "C:\\Program Files\\LLVM\\lib\\clang\\17\\include" -internal-isystem "C:\\Program Files\\Microsoft Visual Studio\\2022\\Professional\\VC\\Tools\\MSVC\\14.38.33130\\include" -internal-isystem "C:\\Program Files\\Microsoft Visual Studio\\2022\\Professional\\VC\\Tools\\MSVC\\14.38.33130\\atlmfc\\include" -internal-isystem "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.22621.0\\ucrt" -internal-isystem "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.22621.0\\shared" -internal-isystem "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.22621.0\\um" -internal-isystem "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.22621.0\\winrt" -internal-isystem "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.22621.0\\cppwinrt" 
-O0 -Wall -Wextra "-fdebug-compilation-dir=C:\\Users\\My Name\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\tinyfiledialogs-3.9.1" -ferror-limit 19 -fno-use-cxa-atexit -fms-extensions -fms-compatibility -fms-compatibility-version=19.38.33135 -fdelayed-template-parsing -faddrsig -o "Z:/dev/foo/project/build/./cargo/build\\x86_64-pc-windows-msvc\\debug\\build\\tinyfiledialogs-beefe8f25dd73cf6\\out\\bd7bb318ec84866d-tinyfiledialogs.o" -x c libtinyfiledialogs/tinyfiledialogs.c
warning: tinyfiledialogs@3.9.1: clang -cc1 version 17.0.6 based upon LLVM 17.0.6 default target x86_64-pc-windows-msvc
warning: tinyfiledialogs@3.9.1: #include "..." search starts here:
warning: tinyfiledialogs@3.9.1: #include <...> search starts here:
warning: tinyfiledialogs@3.9.1:  C:\Program Files\LLVM\lib\clang\17\include
warning: tinyfiledialogs@3.9.1:  C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.38.33130\include
warning: tinyfiledialogs@3.9.1:  C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.38.33130\atlmfc\include
warning: tinyfiledialogs@3.9.1:  C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt
warning: tinyfiledialogs@3.9.1:  C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared
warning: tinyfiledialogs@3.9.1:  C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um
warning: tinyfiledialogs@3.9.1:  C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\winrt
warning: tinyfiledialogs@3.9.1:  C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\cppwinrt
warning: tinyfiledialogs@3.9.1: End of search list.
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:1072:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  1072 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:1145:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  1145 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:1220:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  1220 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:1501:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  1501 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:1607:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  1607 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:1797:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  1797 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:1851:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  1851 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:1920:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  1920 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:1969:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  1969 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:2011:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  2011 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:2067:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  2067 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:2142:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  2142 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:2210:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  2210 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: libtinyfiledialogs/tinyfiledialogs.c:2254:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
warning: tinyfiledialogs@3.9.1:  2254 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
warning: tinyfiledialogs@3.9.1:       |         ^
warning: tinyfiledialogs@3.9.1: 14 errors generated.

error: failed to run custom build command for `tinyfiledialogs v3.9.1`

Caused by:
  process didn't exit successfully: `Z:/dev/foo/project/build/./cargo/build\debug\build\tinyfiledialogs-80268e556e839f44\build-script-build` (exit code: 1)
  --- stdout
  TARGET = Some("x86_64-pc-windows-msvc")
  OPT_LEVEL = Some("0")
  HOST = Some("x86_64-pc-windows-msvc")
  cargo:rerun-if-env-changed=CC_x86_64-pc-windows-msvc
  CC_x86_64-pc-windows-msvc = Some("C:/Program Files/LLVM/bin/clang.exe")
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  cargo:rerun-if-env-changed=CRATE_CC_NO_DEFAULTS
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some("true")
  cargo:rerun-if-env-changed=CFLAGS_x86_64-pc-windows-msvc
  CFLAGS_x86_64-pc-windows-msvc = None
  cargo:rerun-if-env-changed=CFLAGS_x86_64_pc_windows_msvc
  CFLAGS_x86_64_pc_windows_msvc = None
  cargo:rerun-if-env-changed=HOST_CFLAGS
  HOST_CFLAGS = None
  cargo:rerun-if-env-changed=CFLAGS
  CFLAGS = None
  cargo:warning=clang version 17.0.6
  cargo:warning=Target: x86_64-pc-windows-msvc
  cargo:warning=Thread model: posix
  cargo:warning=InstalledDir: C:\Program Files\LLVM\bin
  cargo:warning= (in-process)
  cargo:warning= "C:\\Program Files\\LLVM\\bin\\clang.exe" -cc1 -triple x86_64-pc-windows-msvc19.38.33135 -emit-obj -mrelax-all -mincremental-linker-compatible -disable-free -clear-ast-before-backend -disable-llvm-verifier -discard-value-names -main-file-name tinyfiledialogs.c -mrelocation-model pic -pic-level 2 -mframe-pointer=all -fmath-errno -ffp-contract=on -fno-rounding-math -mconstructor-aliases -funwind-tables=2 -target-cpu x86-64 -tune-cpu generic -gno-column-info -gcodeview -debug-info-kind=constructor -v -ffunction-sections -fdata-sections "-fcoverage-compilation-dir=C:\\Users\\My Name\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\tinyfiledialogs-3.9.1" -resource-dir "C:\\Program Files\\LLVM\\lib\\clang\\17" -D USE_WINDOWS_HIDPI -internal-isystem "C:\\Program Files\\LLVM\\lib\\clang\\17\\include" -internal-isystem "C:\\Program Files\\Microsoft Visual Studio\\2022\\Professional\\VC\\Tools\\MSVC\\14.38.33130\\include" -internal-isystem 
"C:\\Program Files\\Microsoft Visual Studio\\2022\\Professional\\VC\\Tools\\MSVC\\14.38.33130\\atlmfc\\include" -internal-isystem "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.22621.0\\ucrt" -internal-isystem "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.22621.0\\shared" -internal-isystem "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.22621.0\\um" -internal-isystem "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.22621.0\\winrt" -internal-isystem "C:\\Program Files (x86)\\Windows Kits\\10\\Include\\10.0.22621.0\\cppwinrt" -O0 -Wall -Wextra "-fdebug-compilation-dir=C:\\Users\\My Name\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\tinyfiledialogs-3.9.1" -ferror-limit 19 -fno-use-cxa-atexit -fms-extensions -fms-compatibility -fms-compatibility-version=19.38.33135 -fdelayed-template-parsing -faddrsig -o "Z:/dev/foo/project/build/./cargo/build\\x86_64-pc-windows-msvc\\debug\\build\\tinyfiledialogs-beefe8f25dd73cf6\\out\\bd7bb318ec84866d-tinyfiledialogs.o" -x c libtinyfiledialogs/tinyfiledialogs.c
  cargo:warning=clang -cc1 version 17.0.6 based upon LLVM 17.0.6 default target x86_64-pc-windows-msvc
  cargo:warning=#include "..." search starts here:
  cargo:warning=#include <...> search starts here:
  cargo:warning= C:\Program Files\LLVM\lib\clang\17\include
  cargo:warning= C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.38.33130\include
  cargo:warning= C:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.38.33130\atlmfc\include
  cargo:warning= C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\ucrt
  cargo:warning= C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\shared
  cargo:warning= C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\um
  cargo:warning= C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\winrt
  cargo:warning= C:\Program Files (x86)\Windows Kits\10\Include\10.0.22621.0\cppwinrt
  cargo:warning=End of search list.
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:1072:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 1072 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:1145:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 1145 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:1220:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 1220 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:1501:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 1501 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:1607:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 1607 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:1797:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 1797 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:1851:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 1851 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:1920:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 1920 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:1969:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 1969 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:2011:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 2011 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:2067:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 2067 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:2142:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 2142 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:2210:2: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 2210 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=libtinyfiledialogs/tinyfiledialogs.c:2254:9: error: call to undeclared function 'SetProcessDpiAwareness'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  cargo:warning= 2254 |         SetProcessDpiAwareness(PROCESS_PER_MONITOR_DPI_AWARE);
  cargo:warning=      |         ^
  cargo:warning=14 errors generated.

  --- stderr


  error occurred: Command "C:/Program Files/LLVM/bin/clang.exe" "-O0" "--target=x86_64-pc-windows-msvc" "-ffunction-sections" "-fdata-sections" "-g" "-fno-omit-frame-pointer" "-m64" "--target=x86_64-pc-windows-msvc" "-Wall" "-Wextra" "-v" "-DUSE_WINDOWS_HIDPI" "-o" "Z:/dev/foo/project/build/./cargo/build\\x86_64-pc-windows-msvc\\debug\\build\\tinyfiledialogs-beefe8f25dd73cf6\\out\\bd7bb318ec84866d-tinyfiledialogs.o" "-c" "libtinyfiledialogs/tinyfiledialogs.c" with args clang.exe did not execute successfully (status code exit code: 1).



FAILED: CMakeFiles/_cargo-build_cli Z:/dev/foo/project/build/CMakeFiles/_cargo-build_cli
C:\WINDOWS\system32\cmd.exe /C "cd /D Z:\dev\foo\project\rust\cli && "C:\Program Files\CMake\bin\cmake.exe" -E env "CC_x86_64-pc-windows-msvc=C:/Program Files/LLVM/bin/clang.exe" "CXX_x86_64-pc-windows-msvc=C:/Program Files/LLVM/bin/clang++.exe" CORROSION_BUILD_DIR=Z:/dev/foo/project/build "CARGO_BUILD_RUSTC=C:/Users/My Name/.rustup/toolchains/stable-x86_64-pc-windows-msvc/bin/rustc.exe" "C:/Users/My Name/.rustup/toolchains/stable-x86_64-pc-windows-msvc/bin/cargo.exe" rustc --lib --target=x86_64-pc-windows-msvc --package cli --manifest-path Z:/dev/foo/project/rust/cli/Cargo.toml --target-dir Z:/dev/foo/project/build/./cargo/build -- -Cdefault-linker-libraries=yes"
ninja: build stopped: subcommand failed.

```

--- a/build.rs
+++ b/build.rs
@@ -9,7 +9,7 @@ use std::env;
 fn main() {
     let target = env::var("TARGET").unwrap();
 
-    let windows_hidpi = if cfg!(feature = "windows-hidpi") {
+    let windows_hidpi = if cfg!(feature = "windows-hidpi") & !target.ends_with("pc-windows-gnu") {
         "USE_WINDOWS_HIDPI"
     }else {
         "NO_WINDOWS_HIDPI"

